deploy/k8s_maker/templates/azure_k8s_02_misc.yaml

```diff
@@ -0,0 +1,65 @@
+#apiVersion: v1
```

Comment:
Should these lines be deleted?

Reply:
deleted several templates, including this one